nilearn/datasets/tests/test_utils.py

```diff
@@ -387,6 +346,7 @@ def test_fetch_files_overwrite(tmp_path, request_mocker):
 @pytest.mark.skipif(not BOTO_INSTALLED,
                     reason='Boto3  missing; necessary for this test')
 def test_make_fresh_openneuro_dataset_urls_index(tmp_path, request_mocker):
+    import json
```

Comment:
I'll move it to the module level. Note this test is not run on CI (we never have boto3 installed) and the function it tests is never used by the rest of the library. I am wondering if we should just remove it

Reply:
We don't run the test for that on the CI, but it has been created to be run locally, if we ever have to update the function it tests and want a quick to test the changes.

Reply:
I'd lean towards removing it if we don't have it run on the CI, personally.

Reply:
I moved the json imports to the module level and we can discuss whether this function and its test should be removed in a separate issue to avoid cluttering this pr (it's an unrelated issue)